### ISSUE TITLE
[BUG] ADD A TITLE HERE

### ISSUE THREAD

**Issue by 9ghtX** [Fevol/obsidian-translate#69]
# User report
**Description:** After installation of Bergamot and FastText the "Translate Selection (default language) doesn't work. Error: "Automatic language detection is not supported (400)".

---
# Debugger data (do not alter)
**plugin_version**: "1.4.7"
**platform**: "Desktop"
**framework_version**: "1.5.3"
**obsidian_version**: "1.5.11"
**selected_service**: "bergamot"

**Comment by 9ghtX** [comment]
Yes, after restart it still throwing this message. I also tried reinstalling plugin, changing translate service (tried bergamot and lingva)

**Comment by merlinuwe** [comment]
I have the same issue.  
I would like to test if there is a new version.

SYSTEM INFO:
	Obsidian version: v1.6.7
	Installer version: v1.6.7
	Operating system: Windows 10 Home 10.0.19045
	Login status: not logged in
	Insider build toggle: off
	Live preview: off
	Base theme: dark
	Community theme: MagicUser v20.8.0
	Snippets enabled: 43
	Restricted mode: off

**Comment by VLTNOgithub** [comment]
Was this ever fixed? Still happening to me

**Comment by tonperre** [comment]
Happens to me too. Windows 11, Obsidian 1.8.9. 

I only tested with bergamot
